tests/test_toa_reader.py

```diff
@@ -52,3 +77,27 @@ def test_jump_3(self):
 
     def test_obs(self):
         assert self.x.table[1]["obs"] == "gbt"
+
+    def test_ephem(self):
+        assert self.x.ephem == "DE421"
+
+    def test_planets(self):
+        assert self.x.planets is False
+
+    def test_clock(self):
+        assert self.x.clock_corr_info["bipm_version"] == bipm_default
+
+    def test_model_override1(self):
+        parstr = StringIO(simplepar)
+        m = get_model(parstr)
+        y = toa.get_TOAs("test1.tim", model=m)
+        assert y.ephem == "DE436"
+        assert y.planets is True
+        assert y.clock_corr_info["bipm_version"] == "BIPM2017"
+
+    def test_model_override2(self):
```

Comment:
Leaving these functions inside TestTOAReader means that the setup function, which loads some TOAs, is done for each of them. Just unindenting them so they're bare test functions will get them run without the class-based setup. (Classes are only needed for `unittest`.)

Reply:
Ah.  Didn't know that.  Cool.